packages/react/src/separator/Separator.tsx

```diff
@@ -0,0 +1,27 @@
+import * as RadixSeparator from "@radix-ui/react-separator";
+import { Slot } from "@radix-ui/react-slot";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import { Box } from "../box";
+import { type ExtendProps } from "../utils";
+import { type Recipe, recipe } from "./Separator.recipe";
+
+type SeparatorProps = ExtendProps<
+  ComponentPropsWithRef<typeof RadixSeparator.Root>,
+  ComponentPropsWithRef<typeof Box>,
+  Recipe
+>;
+
+export const Separator = forwardRef<HTMLHRElement, SeparatorProps>(
+  ({ asChild, ...props }, ref) => {
+    const Comp = asChild ? Slot : RadixSeparator.Root;
+
+    return (
+      <Box asChild ref={ref} {...props} {...recipe(props)}>
+        <Comp />
+      </Box>
+    );
```

Comment:
i don't think we need to use `Slot` here - it's a single empty element so consumers will probly never need to swap it out for something else
```suggestion
    return (
      <Box asChild ref={ref} {...props} {...recipe(props)}>
        <RadixSeparator.Root />
      </Box>
    );
```

Reply:
Done in https://github.com/optimizely-axiom/optiaxiom/pull/65/commits/7058c9cb762aad5efd9dc79ad3dcaeae06c45a9a.